packages/storybook-react/src/stories/ButtonLink/Button.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { ButtonLink } from '@utrecht/component-library-react';
+import React from 'react';
+import { argTypes } from './argTypes';
+
+export default {
+  title: 'React.js Component/ButtonLink',
+  id: 'react-button-link',
+  component: ButtonLink,
+  argTypes,
+} as ComponentMeta<typeof ButtonLink>;
+
+const Template: ComponentStory<typeof ButtonLink> = (args) => <ButtonLink {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  children: 'Read more...',
+};
```

Comment:
The files in `ButtonLink/` are called `Button*.tsx` instead of `ButtonLink*.tsx`